graphql-cli/src/test/resources/serviceGen/expectedServices/typesWithMultiDimensionalListsDefault.bal

```diff
@@ -0,0 +1,22 @@
+import ballerina/graphql;
```

Comment:
The expected `.bal` files don't seem to be formatted correctly. We might need to format them and fix the code generation formatting.

Reply:
I formatted the expected `.bal` files. Generated code doesn't contain these formatting issues.

Reply:
Shall we check the `readContent(..)` method ? it seems like it removes all occurrences of consecutive whitespace characters. We may need to compare these two files without removing the white spaces to check the formatting.

Reply:
Sure, I'll change and fix it.